_posts/2018-02-27-optimizing-grpc-part-1.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Optimizing gRPC - Part 1
+published: false
+permalink: blog/optimizing-grpc-part-1
+author: Carl Mastrangelo
+company: Google
+company-link: https://www.google.com
+---
+
+# Optimizing gRPC - Part 1
+
+A common question with gRPC is how to make it fast.  The gRPC library offers users access to high
+performance RPCs, but it isn't always clear how to do it.   Because this question is common enough
```

Comment:
suggest: "...but it isn't always clear how to achieve this"

Reply:
Done